pytest.ini

```diff
@@ -0,0 +1,14 @@
+[pytest]
```

Comment:
Could you provide a description of pytest (or put links where to read)? I think it would be helpful to put into a dedicated section in the repo's README information about:

- How to run pytest and how to integrate it into the common git work
- How to extend the set of test cases

Reply:
[pytest](https://docs.pytest.org/en/latest/) is probably the most popular python testing framework,
both easy to use and versatile. It's supported in all the IDE's I've used for Python so far: Eclipse, PyCharm, VSCode. The major scientific and math libraries Numpy, Scipy also use it.
You can add some test_smth.py file to the tests folder and add there some test_smth() method. Ordinary builtin asserts are redefined and produce information like much more complicated assertions from Unittests for example.